cts-java/src/test/java/org/ga4gh/cts/api/references/ReferenceSetsSearchIT.java

```diff
@@ -21,49 +21,132 @@
-        final int ncbiTaxonId = 9606;
-        final String assemblyId = "GRCh38";
-
-        final SearchReferenceSetsRequest req =
-                SearchReferenceSetsRequest.newBuilder()
-                                            .setAccessions(aSingle(accessionNumber))
-                                            .build();
+    public void checkSearchingReferenceSetsReturnsSome() throws AvroRemoteException {
+        final SearchReferenceSetsRequest req = 
+            SearchReferenceSetsRequest.newBuilder().build();
         final SearchReferenceSetsResponse resp = client.references.searchReferenceSets(req);
+
+        final List<ReferenceSet> sets = resp.getReferenceSets();
+        assertThat(sets).isNotEmpty();
+        sets.stream().forEach(rs -> assertThat(rs.getId()).isNotNull());
+    }
+
+
+    /**
+     * Fetch reference sets and make sure they're well-formed.
+     *
+     * @throws AvroRemoteException if there's a communication problem or server exception ({@link GAException})
+     */
+    @Test
+    public void checkSearchingReferenceSetsReturnsWellFormed() throws AvroRemoteException {
+        final SearchReferenceSetsRequest req = 
+            SearchReferenceSetsRequest.newBuilder().build();
+        final SearchReferenceSetsResponse resp = client.references.searchReferenceSets(req);
+
+        final List<ReferenceSet> sets = resp.getReferenceSets();
+
+        sets.stream().forEach(rs -> assertThat(rs.getId()).isNotNull());
+        sets.stream().forEach(rs -> assertThat(rs.getMd5checksum()).isNotNull());
+        sets.stream().forEach(rs -> assertThat(rs.getSourceAccessions()).isNotNull());
+    }
+
+    /**
+     * Search for reference set by accession number.
+     *
+     * @throws AvroRemoteException if there's a communication problem or server exception ({@link GAException})
+     */
+    @Test
+    public void searchForReferenceSetByAccessionNumber() throws AvroRemoteException {
+        final String accession = "GA4GH_CTS_01";
+        final String assemblyId = "hg37";
+
+        final SearchReferenceSetsRequest req = 
+            SearchReferenceSetsRequest.newBuilder()
+                                        .setAccessions(aSingle(accession))
+                                        .build();
+        final SearchReferenceSetsResponse resp = client.references.searchReferenceSets(req);
+        
         final List<ReferenceSet> refSets = resp.getReferenceSets();
+        assertThat(refSets).isNotEmpty();
+        refSets.stream().forEach(rs -> assertThat(rs.getAssemblyId()).isEqualTo(assemblyId));
+    }
 
-        refSets.stream().forEach(rs -> assertThat(rs.getNcbiTaxonId()).isEqualTo(ncbiTaxonId));
+    /**
+     * Search for reference set by md5 checksum.
+     *
+     * @throws AvroRemoteException if there's a communication problem or server exception ({@link GAException})
+     */
+    @Test
+    public void searchForReferenceSetByMD5Checksum() throws AvroRemoteException {
```

Comment:
I haven't been terribly consistent in this, but let's make sure these test methods have names that describe what we're looking for.  In this case, something like `checkFetchingReferenceSetsByChecksum` would be good.